src/xml2js.coffee

```diff
@@ -402,6 +402,23 @@ class exports.Parser extends events.EventEmitter
       else if @saxParser.ended
         throw err
 
+  parseStringSync: (str) =>
```

Comment:
Can you make the API adhere to the one of the regular `parseString` and also add the shorthand alias `exports.parseStringSync`?